contrib/docker/README

```diff
@@ -0,0 +1,35 @@
+Avocado-vt inside the container
+
+That a simple way to use avocado-vt in the system when you are worried
+about breaking the system or you wish to test a new version of the
+framework.
+
+How to use avocado-vt inside the container:
+First, you need clone the dockerfile Fedora.dockerfile
+The Fedora-30 uses podman as a default container engine instead of docker
+
+Build an image:
+
+# podman build --build-arg avocado_version=74.0 -t avocado-vt -f Fedora.dockerfile .
+
+To change avocado_version or Fedora distro use next args:
+            --build-arg avocado_version=74.0
+            --build-arg TAG_VERSION=30
+
+To run tests need to add a list of volumes from the system and the list
+will be depends-on which test you want to run:
+
+To run the tests need to add a list of volumes from the system.
+The list will be depends-on which test you want to run, maybe you need to
+provide some specific utils:
+
+avocado_command - test command
+
+Example:
+#podman run --name avocado --network host --rm --privileged -v "/usr/share/qemu/:/usr/share/qemu/" \
```

Comment:
This won't work unless Avocado has been bootstrapped in the container image.  I suggest adding:

```
RUN avocado vt-bootstrap --yes-to-all --vt-skip-verify-download-assets
```